src/components/App.js

```diff
@@ -2144,7 +2144,7 @@ class App extends Component {
                     .split(/\s*info\s+/).slice(1)
                     .map(x => x.trim())
                     .map(x => {
-                        let match = x.match(/[A-Za-z]\d+(\s+[A-Za-z]\d+)*$/)
+                        let match = x.match(/(pass|[A-Za-z]\d+)(\s+(pass|[A-Za-z]\d+))*$/)
```

Comment:
In addition to include `pass` in the Regex, we also need to filter out all PV which contains `pass`, because `board.coord2vertex()` can't handle `'pass'` as argument. In line 2166, we should do something like 

~~~js
.filter(({move, pv}) => move.match(/^[A-Za-z]\d+$/) && !pv.includes('pass'))
~~~

Reply:
After thinking on it, its better to trim the pv from pass moves onwards in the regex section.  That way we keep analyzing variations that are near the end of the game.